bxml/buildingBlocks/createCall.md

```diff
@@ -0,0 +1,23 @@
+{% method %}
+# Create an Outbound Call
```

Comment:
For [credentials](https://github.com/Bandwidth/ap-docs/pull/101/files#diff-867f3a8f60725cab4c74ed9632cb5b3cR3) we used `##` instead of `#`. I, too, make this mistake a lot. Pick either H1 (`#`) or H2 (`##`) and be consistent.

Reply:
I am using an H1 for the main title and H2 for subtitles